src/metal.cs

```diff
@@ -1457,6 +1455,21 @@ public partial interface MTLRenderCommandEncoder : MTLCommandEncoder {
 
 		[Abstract, Export ("setVisibilityResultMode:offset:")]
 		void SetVisibilityResultMode (MTLVisibilityResultMode mode, nuint offset);
+		
+		[iOS (10,0), TV (10,0), NoWatch, Mac (10,12)]
+		[Abstract]
```

Comment:
same issue with those new `[Abstract]` - they are being added to an existing type